packages/components/src/Combobox/Combobox.css

```diff
@@ -0,0 +1,12 @@
+.overlay {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: var(--elevation-menu);
+
+  @media (--small-screens-and-up) {
+    background-color: transparent;
```

Comment:
Is there a reason I am missing as to why this isn't needed on all screens? I assume I am missing something but wanted to be sure

Reply:
ah good catch. I pulled that same overlay styling from another instance where it uses a darker opacity on very small screens, but I deleted those styles and forgot to adjust this bit

given the state it is in right now, you're right I'll move that out of the media query and apply it all the time